pyproject.toml

```
@@ -13,7 +13,7 @@ repository = "https://github.com/rob-blackbourn/httpx-negotiate-sspi"
 [tool.poetry.dependencies]
 python = "^3.8"
 pywin32 = ">=223"
-httpx = "> 0.16, < 0.28"
+httpx = "> 0.16, <= 0.28.1"
```

It's usually not a good idea to pin to a patch version as it means you won't pick up even security fixes.
```suggestion
httpx = ">0.16,<=0.29"
```